tracer/build/_build/Build.GitHub.cs

```diff
@@ -422,9 +431,17 @@ partial class Build
 
             Console.WriteLine("Release notes generated");
 
-            static (string category, Issue issue) CategoriseIssue(Issue issue)
+            static (string category, Issue issue) CategorizeIssue(Issue issue)
             {
                 var fixIssues = new[] { "type:bug", "type:regression", "type:cleanup" };
+                var areaLabelToComponentMap = new Dictionary<string, string>() {
+                    { "area:tracer", tracer },
+                    { "area:ci-app", ciApp },
+                    { "area:app-sec", appSec },
+                    { "area:profiler", profiler },
+                    { "area:debugger", debugger }
```

Comment:
If we do this, I think we should bring back automatic labelling for the `area:*` tags. Even if it's not perfect, it will be better than nothing

Reply:
Looking back at this, I'm not sure if we should bring back the labeller as it existed. If we do this based on file path, I feel that it will be over labelling :) So I think we should have a labeller based:
- on path for the profiler, maybe the debugger as well. 
- on PR Title for the components in Tracer (I feel that we are quite good at specifying appsec, ciapp, serverless in the title). WDYT? (cc @lucaspimentel would could have an opinion as well)